### ISSUE TITLE
Cannot use namespace 'EventEmitter' as a type.

### ISSUE THREAD

**Issue by YpsilonTM** [obs-websocket-community-projects/obs-websocket-js#335]
So this is my code and I keep getting the error that EventEmitter cannot be used as a type.
I remove my node modules folder did a clean install. Anyone have a clue why it keeps complaining about that?

```js
import OBSWebSocket from 'obs-websocket-js'
import { OBS_SETTINGS } from '../config.js'

class OBS {
  private socketURL: string
  private socket: OBSWebSocket

  constructor() {
    this.socketURL = OBS_SETTINGS.getSocketURL()
  }

  public async connect() {
    this.socket = new OBSWebSocket()
    return await this.socket.connect(this.socketURL)
  }
}
```

The Error:

```bash
node_modules/obs-websocket-js/dist/base.d.ts:23:56 - error TS2507: Type 'typeof import("D:/Development/silon-server-twitch/node_modules/eventemitter3/index")' is not a constructor function type.

23 export declare abstract class BaseOBSWebSocket extends EventEmitter<MapValueToArgsArray<EventTypes>> {
                                                          ~~~~~~~~~~~~

node_modules/obs-websocket-js/dist/base.d.ts:27:34 - error TS2709: Cannot use namespace 'EventEmitter' as a type.

27     protected internalListeners: EventEmitter<string | symbol, any>;
                                    ~~~~~~~~~~~~


Found 2 errors in the same file, starting at: node_modules/obs-websocket-js/dist/base.d.ts:23
```

### Versions Used:
  - obs-websocket-js version: 5.0.3
  - node version: v18.14.1

**Comment by YpsilonTM** [comment]
"skipLibCheck": true,

Just ignore the lib errors forgot about that.

**Comment by YpsilonTM** [comment]
So it actually now fixed.

**Comment by YpsilonTM** [comment]
in `node_modules\obs-websocket-js\dist\base.d.ts` adjusting
`import EventEmitter from 'eventemitter3';`
to
`import { EventEmitter } from 'eventemitter3';`

I saw its exported as default and named. So gave that a try still don't get why it then works.

fixed all the typescript complaining for me. And I have my types back.

**Comment by Hoishin** [comment]
> in `node_modules\obs-websocket-js\dist\base.d.ts` adjusting
> 
> `import EventEmitter from 'eventemitter3';`
> 
> to
> 
> `import { EventEmitter } from 'eventemitter3';`
> 
> 
> 
> I saw its exported as default and named. So gave that a try still don't get why it then works.
> 
> 
> 
> fixed all the typescript complaining for me. And I have my types back.

This is already fixed in the repo but has not been published yet.

**Comment by YpsilonTM** [comment]
Ok thank you maybe one day I will see the next release.

**Comment by mint-dewit** [comment]
@Hoishin is it possible to get this published? it is quite annoying.

**Comment by Hoishin** [comment]
@mint-dewit I'm not maintainer of this repo/package

**Comment by t2t2** [comment]
Published 5.0.4 which includes #333
